### PR TITLE
add guru: class that gives hints about data

### PR DESCRIPTION
**Pull request by onaga1958**

### CODE REVIEW

**Comment by onaga1958** [comment]
[эдвайзер категориальных фичей](https://trello.com/c/7phHoz75/12-%D1%8D%D0%B4%D0%B2%D0%B0%D0%B9%D0%B7%D0%B5%D1%80-%D0%BA%D0%B0%D1%82%D0%B5%D0%B3%D0%BE%D1%80%D0%B8%D0%B0%D0%BB%D1%8C%D0%BD%D1%8B%D1%85-%D1%84%D0%B8%D1%87%D0%B5%D0%B9)

**Comment by anaderi** [comment]
1. make it simpler (like chat-bot interface: Guru.check_class_balance(y))
2. move all params to __init__
3. all keys in internal dicts -> const
4. messages -> dict
5. add Guru.check_correlation(X, f1, f2) - to plot 2D histogram f1 over f2

File: tests/test_guru.py [hunk 1]
```diff
@@ -1,61 +1,63 @@
...
+
+def test_class_disbalance():
+    guru_params = [{}, {}, {'class_disbalance_qoute': 0.4},
+                   {'class_disbalance_qoute': 0.8}]
+    args = [[_BREAST_Y]] + [[_TOY_Y]] * 2
+    answers = [{},
+               {Guru._TOO_COMMON_KEY: [0], Guru._TOO_RARE_KEY: [1]},
+               {Guru._TOO_RARE_KEY: [1]},
+               {Guru._TOO_COMMON_KEY: [0], Guru._TOO_RARE_KEY: [1, 3]}]
+    _test(lambda guru, args: guru.check_class_disbalance(*args), guru_params, args, answers)
```
Comment (anaderi) [inline tests/test_guru.py]: why do you need lambda here? can't you just pass `guru.check_class_disbalance`?

Comment (onaga1958) [reply tests/test_guru.py]: I need different Guru objects to test different parameters of Guru.__init__

Comment (anaderi) [reply tests/test_guru.py]: why can't you init Guru once before _test?

**Pull request merged**